crawl-ref/source/player.cc

```diff
@@ -3916,6 +3915,10 @@ int get_real_hp(bool trans, bool drained)
             hitp = hitp * 3 / 2;
     }
 
+    //Divine vigour should apply multiplicatively with all other effects
```

Comment:
Don't think these comments are needed. If we wanted something like this, we'd note `// The below effects multiply each other, rather than applying additively` - but I think it's pretty clear from context. We definitely don't need a comment just for divine vigour.

Also, put a space after the `//`.